app/ledger.js

```diff
@@ -129,8 +129,15 @@ const doAction = (action) => {
       break
 
     case appConstants.APP_CHANGE_SETTING:
-      if (action.key === settings.PAYMENTS_ENABLED) return initialize(action.value)
-      if (action.key === settings.PAYMENTS_CONTRIBUTION_AMOUNT) return setPaymentInfo(action.value)
+      switch (action.key) {
+        case settings.PAYMENTS_ENABLED:
+          return initialize(action.value, 'changeSettingPaymentsEnabled')
```

Comment:
probably better to get rid of `return` and add a `break` line -- that would be consistent with everything else in the function.